spec/system/jobseekers/jobseekers_can_add_qualifications_to_their_profile_spec.rb

```diff
@@ -18,7 +18,8 @@
       before { click_on "Add qualifications" }
 
       it "passes a11y", :a11y do
-        expect(page).to be_axe_clean.skipping "region", "landmark-no-duplicate-banner"
+        # area-expanded on input
```

Comment:
[Minor] I don't understand what this comment means, but could be just my lack of knowledge/context on axe.